sdks/python/apache_beam/io/lyft/kinesis.py

```diff
@@ -20,6 +20,7 @@ class FlinkKinesisInput(PTransform):
   stream = None
   encoding = None
   max_out_of_orderness_millis = None
+  use_watermark_tracker = None
```

Comment:
if this is a flag to use the JobManagerWatermarkTracker, I would recommend to rename the variable to `use_global_watermark_tracker` or `use_job_manager_watermark_tracker`